webpush-fcm-relay.go

```diff
@@ -33,9 +33,7 @@ var (
 )
 
 func main() {
-	tracer.Start(
-		tracer.WithService("webpush-fcm-relay"),
```

Comment:
Are you removing this because it is set in the deployment through environment variables?

Reply:
Yes, this value is picked up through `DD_*` variables already.